optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -875,7 +872,7 @@ This test verifies that after solving, all lessons are assigned to a time slot a
 It also verifies that it found a feasible solution (no hard constraints broken).
 
 Normally, the solver finds a feasible solution in less than 200 milliseconds.
-Notice how the `@SpringBootTest` annotation's `properties` overwrites the solver termination
+Notice how the `@SpringBootTest` annotation's properties overwrites the solver termination
```

Comment:
Given that the code states `@SpringBootTest(properties = {...}`, I presume this is more correct?

```suggestion
Notice how the `@SpringBootTest` annotation's `properties` property overwrites the solver termination
```

Reply:
Yes. I will fix when we agree on timetable/ time table.